www/scripts/codecheckerviewer/filter/DetectionStatusFilter.js

```diff
@@ -0,0 +1,88 @@
+// -------------------------------------------------------------------------
+//                     The CodeChecker Infrastructure
+//   This file is distributed under the University of Illinois Open Source
+//   License. See LICENSE.TXT for details.
+// -------------------------------------------------------------------------
+
+define([
+  'dojo/dom-construct',
+  'dojo/dom-class',
+  'dojo/_base/declare',
+  'dojo/Deferred',
+  'codechecker/filter/SelectFilter',
+  'codechecker/util'],
+function (dom, domClass, declare, Deferred, SelectFilter, util) {
+  return declare(SelectFilter, {
+    notAvailableMsg : 'Not available because uniqueing is enabled!',
```

Comment:
I'm thinking about a bit more descriptive message here. What do you think?
`Not available in uniqueing mode! Several detection statuses could belong to the same bug!`